src/components/Draft/DraftList.jsx

```diff
@@ -1,79 +1,66 @@
-﻿/* eslint-disable react/no-unused-state */
-import React, { Component } from "react";
-import { connect } from "react-redux";
+﻿import React, { useEffect } from "react";
 import { createBrowserHistory } from "history";
 import PropTypes from "prop-types";
-import DraftEventCard from "./DraftEventCard";
 import RenderList from "../Event/RenderList/RenderList";
-import { changeEventStatus } from "../../actions/event/event-item-view-action";
-import { EVENT_STATUS_ENUM } from "../../constants/eventConstants";
-import { setSuccessAllert } from "../../actions/alert-action";
+import { ALERT_MESSAGES } from "../../constants/draftConstants";
+import filterHelper from "../helpers/filterHelper";
 
 const history = createBrowserHistory({ forceRefresh: true });
 
-class DraftList extends Component {
-  constructor() {
-    super();
-    this.state = {
-      currentPage: 1,
-    };
-  }
+const DraftList = ({ events, deleteEvent, alert, getDraftsAction }) => {
```

Comment:
where you get `events`?? Why don't you get them from container -> state?

Reply:
it comes from container